src/validate.js

```diff
@@ -53,13 +63,16 @@ function toErrorSchema(errors) {
       }
       parent = parent[segment];
     }
+
     if (Array.isArray(parent.__errors)) {
       // We store the list of errors for this node in a property named __errors
       // to avoid name collision with a possible sub schema field named
       // "errors" (see `validate.createErrorHandler`).
       parent.__errors = parent.__errors.concat(message);
     } else {
-      parent.__errors = [message];
+      if (message) {
```

Comment:
@aerfio when would `message` be undefined?

Reply:
If there is no `message` field in particular error, like in that special case I've introduced
If I'd make validation error like 
```
{
   stack: something,
   message: something
}
```
then it would look like this:
![image](https://user-images.githubusercontent.com/17271979/53367029-5b5f2700-3945-11e9-9c88-b502df302af0.png)
Which is bad because we have the same info in errorList and in the footer

Reply:
@epicfaace Error thrown by ajv has only name and stack fields, name is `Error`, stack is like in pic above

Advantage of using message field is that while form has validation errors, then all fields are red, opposite to https://github.com/mozilla-services/react-jsonschema-form/pull/1130#issuecomment-467088376